quickstart/maven.markdown

```diff
@@ -335,19 +335,19 @@ Defaults to false.
 
 ### timestampedReports 
 
-By default PIT will create a date and time stamped folder for its output each it is run. This can can make automation difficult, so the behaviour can be suppressed by setting timestampedReports to false.
+By default PIT will create a date and time stamped folder for each output each it runs. This can make automation difficult, so the behaviour can be suppressed by setting timestampedReports to false.
```

Comment:
```suggestion
By default, PIT will create a date and time stamped folder for each output each it runs. This can make automation difficult, so the behaviour can be suppressed by setting timestampedReports to false.
```